routes/users.js

```diff
@@ -2,51 +2,38 @@ var express = require("express");
 var router = express.Router();
 var db = require("../db.js");
 
-router.route("/")
-  .get(function (req, res) {
-    let user = req.body;
-    db.users.getUserByName(user).then(user => {
-      if (!user) {
-        res.statusCode = 404;
-        res.end()
-        return;
-      }
-      res.statusCode = 200;
-      res.json(user);
-    });
-  })
-  .post(function (req, res) {
-    let user = req.body;
-    db.users.insert(user).then(([result, fields]) => {
-      if (result.affectedRows === 0) {    
-        res.statusCode = 404;
-        res.end();
-        return;
-      }
-       res.statusCode = 200;
-       res.json({ "id": result.insertId })
-    });
-  })
+router.route("/").get(function (req, res) {
+  let user = req.body;
+  db.users.getUserByName(user).then((user) => {
+    if (!user) {
+      res.statusCode = 404;
+      res.end();
+      return;
+    }
+    res.statusCode = 200;
+    res.json(user);
+  });
+});
```

Comment:
Again, most of this is prettier's doing. The real change here is that I removed the POST /users route because new users are created via the /login route